package.json

```diff
@@ -38,9 +38,10 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "eslint": "eslint .",
-    "eslint:fix": "eslint --fix .",
-    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\"",
-    "scsslint": "sass-lint-auto-fix"
+    "eslint:fix": "eslint --fix --max-warnings=0",
+    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|md)\"",
+    "scss-lint": "sass-lint 'src/**/*.scss' -v -f table --max-warnings 328",
```

Comment:
Also, any reason why `--max-warnings` is set to 328? Is that the number of warnings we are getting at the moment?

Reply:
Yes, @jeremyphilemon  is working on reducing the number